app/views/wishlists/show.html.erb

```diff
@@ -1,14 +1,28 @@
-<h1><%= @wishlist.name %></h1>
+<div class="container">
+  <h1 class="wishlist-name"><%= @wishlist.name %></h1>
 
-<p><%= @wishlist.description %></p>
+  <p class="wishlist-description"><%= @wishlist.description %></p>
 
-<p>
-  <strong>Your unique link for this whislist:</strong><br>
-  <%= wishlists_for_guest_url(@wishlist.token) %>
-</p>
+  <div class="input-group wishlist-url-display input-group-sm">
+    <span class="input-group-addon" id="sizing-addon2">Link for sharing:</span>
+    <input type="text" class="form-control" id="unique_url" value="<%= wishlists_for_guest_url(@wishlist.token) %>" aria-describedby="sizing-addon1">
+    <span class="input-group-addon btn" id="sizing-addon2" data-clipboard-action="copy" data-clipboard-target="#unique_url">copy</span>
+  </div>
 
-<h2>Gifts:</h2>
-  <ul>
-    <%= render partial: "item", collection: @wishlist.items %>
-  </ul>
-<%= link_to "Back", wishlists_path %>
+  <div class="wishlist-main-buttons">
+    <a href="#" class="btn btn-default">
+      <span class="glyphicon glyphicon-plus"></span> Add a gift
+    </a>
+    <a href="<%= edit_wishlist_path(@wishlist) %>" class="btn btn-default">
+      <span class="glyphicon glyphicon-edit"></span> Edit list description
+    </a>
+    <%= link_to @wishlist, method: :delete, class: "btn btn-default" do %>
+      <span class="glyphicon glyphicon-remove"></span> Delete this list
+    <% end %>
+  </div>
+
+  <div class="container wishlists-row">
+  <%= render partial: "item", collection: @wishlist.items %>
+  </div>
+
```

Comment:
Eh, dunno. Usually I don't leave an empty line on the closing tags. We should find a style guide and add a linter for HTML and SCSS.